Marlin/src/module/probe.cpp

```diff
@@ -556,6 +556,17 @@ float Probe::run_z_probe(const bool sanity_check/*=true*/) {
     float probes[TOTAL_PROBING];
   #endif
 
+  #if ENABLED(STRAIN_GAUGE_PROBE) && PINS_EXIST(OPTO_SWITCH, COM)
```

Comment:
I've found it works best to tare the probe in `try_to_probe`:

https://github.com/CR6Community/Marlin/commit/e80d24a38cd64ee3d8c2275404d28e9058c4ff0e in combination with https://github.com/CR6Community/Marlin/commit/93ba1b1067a1f7faaca020177ef2d5026c385005

Note that for improved accuracy, the speed at which leveling happens is also lowered.

Reply:
Speed of probing / homing can be tuned in the configuration.

Reply:
> Speed of probing / homing can be tuned in the configuration.

Of course, but I think that a lower speed is a more sensible default for the CR-6 Configuration.h 😉